packages/common-ui/src/helptips/OnboardingTips.test.tsx

```diff
@@ -62,5 +60,12 @@ describe('OnboardingTips Components', () => {
         expect(view).toMatchSnapshot();
       });
     });
+    [DevicesAcceptedOnboarding, GetStartedTip].forEach(async Component => {
+      it(`renders ${Component.displayName || Component.name} correctly`, () => {
+        const { baseElement } = render(<Component />);
+        const view = baseElement.firstChild.firstChild;
+        expect(view).toMatchSnapshot();
+      });
+    });
```

Comment:
why are these two separated from the rest?

Reply:
Otherwise createdGroup="testgroup" is passed as an attribute to DOM and causing warnings: 
https://github.com/NorthernTechHQ/nt-gui/pull/190/files/f737f42d0423059e36032127b4f13fffe9ec34c9..6f9d4d4b45ad30487dfcb3f79d7f6ba63bfe56d1#diff-e2d4ffce7ee62bf48bb2cae01e90817cbe16c41e7d5b8981a05f26d7ab771cf3L163

Reply:
I thought so, but here it's not apparent